### PR TITLE
Update repository.md

### PR DESCRIPTION
**Pull request by Gappa**
Docs update - #324

### CODE REVIEW

File: docs/en/repository.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Not anymore! Kdyby adds possibility to search by relation properties.
 $commentsRepository->findBy(['article.title' => $title]);
 ```
 
+Using `like` is also possible:
+
+```php
+$commentsRepository->findBy(['article.title like' => '%' . $title . '%']);
```
Comment (enumag) [inline docs/en/repository.md]: add like to the listed operations just below instead of here

Comment (Gappa) [reply docs/en/repository.md]: I was deciding between the two and chose the wrong one :)

Hopefully I've updated the PR correctly.

File: docs/en/repository.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ Those operators are:
 - `<=`
 - `>`
 - `>=`
+- `like`
```
Comment (enumag) [inline docs/en/repository.md]: add the like example to the other operation examples below ;-)

Comment (Gappa) [reply docs/en/repository.md]: And now? :)

**Comment by enumag** [comment]
Merged, thanks. ;-)

**Comment by Gappa** [comment]
Thanks for the patience - I know that reviewing/merging PRs can sometimes take longer than doing it yourself :))

**Pull request merged**